### PR TITLE
Correct comment

### PR DESCRIPTION
**Pull request by johnnymo87**
#### Purpose
Make the comment reflect what the function does

#### Changes
Correct comment

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
Nice find! Would you like to add a changelog to take credit for the correction?

File: lib/active_model_serializers/serializable_resource.rb [hunk 1]
```diff
@@ -49,7 +49,7 @@ def serializer_instance
 
     # Get serializer either explicitly :serializer or implicitly from resource
     # Remove :serializer key from serializer_opts
-    # Replace :serializer key with :each_serializer if present
+    # Replace :each_serializer key with :serializer if present
```
Comment (bf4) [inline lib/active_model_serializers/serializable_resource.rb]: Hmm, I don't think it's exactly clear either way, now that I look at it.  Should be

> Remove `:each_serializer` if present and set as `:serializer` key

Comment (bf4) [inline lib/active_model_serializers/serializable_resource.rb]: (or something like that)

**Comment by johnnymo87** [comment]
@NullVoxPopuli @bf4 updated

**Comment by bf4** [comment]
@johnnymo87 Merged. Sorry I missed this was ready.

**Pull request merged**